examples/jaeger/src/main/java/io/opentelemetry/example/JaegerExample.java

```diff
@@ -0,0 +1,80 @@
+package io.opentelemetry.example;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.opentelemetry.exporters.jaeger.JaegerGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.TracerSdkFactory;
+import io.opentelemetry.sdk.trace.export.SimpleSpansProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
+import io.opentelemetry.trace.TracerFactory;
+
+public class JaegerExample {
+  // Jaeger Endpoint URL and PORT
+  private String ip; // = "jaeger";
+  private int port; // = 14250;
+  // OTel API
+  private Tracer tracer;
+  private TracerSdkFactory tracerFactory;
+  // Create a channel towards Jaeger end point
+  private ManagedChannel jaegerChannel;
+  // Export traces to Jaeger
+  private JaegerGrpcSpanExporter jaegerExporter;
+
+  private void setupJaegerExporter() {
+    // Create a channel towards Jaeger end point
+    this.jaegerChannel = ManagedChannelBuilder.forAddress(ip, port).build();
+    // Export traces to Jaeger
+    this.jaegerExporter =
+        JaegerGrpcSpanExporter.newBuilder()
+            .setServiceName("example")
+            .setChannel(this.jaegerChannel)
+            .setDeadline(10)
+            .build();
+    
+    // Set to process the spans by the Jaeger Exporter
+    this.tracerFactory = OpenTelemetrySdk.getTracerFactory();
+    this.tracerFactory.addSpanProcessor(SimpleSpansProcessor.newBuilder(this.jaegerExporter).build());
+  }
+
+  private void myWonderfulUseCase() {
+    // Give a name to the tracer
+    this.tracer = tracerFactory.get("io.opentelemetry.example.JaegerExample");
```

Comment:
Move this to line 18?

Reply:
In bf9797f, I moved the initialization of `tracer` and `tracerFactory` into the constructor.